<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleTrackerWindow</class>
 <widget class="QMainWindow" name="PeopleTrackerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>People Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="colorMappingCheckBox">
         <property name="text">
          <string>Color</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveMeshPushButton">
         <property name="text">
          <string>SaveMesh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pointCloudPushButton">
         <property name="text">
          <string>PointCloud</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="trianglePushButton">
         <property name="text">
          <string>Triangles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPlaneCheckBox">
         <property name="text">
          <string>Show plane</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="resetCamera">
         <property name="text">
          <string>Reset camera</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="setAsBackgroundButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Set as background</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadParametersButton">
         <property name="text">
          <string>Load parameters</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="ntk::MeshViewer" name="mesh_view" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </widget>
       </item>
       <item>
        <widget class="PeopleTrackerImageWidget" name="image_view" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ntk::MeshViewer</class>
   <extends>QWidget</extends>
   <header location="global">ntk/mesh/mesh_viewer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PeopleTrackerImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">PeopleTrackerWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
